src/KurrentDB.NETCore.Compatibility/KurrentDB.NETCore.Compatibility.csproj

```diff
@@ -1,13 +1,13 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
 		<GenerateAssemblyInfo>true</GenerateAssemblyInfo>
 	</PropertyGroup>
 	<ItemGroup>
-		<PackageReference Include="System.ServiceModel.Http" Version="6.2.0" />
+		<PackageReference Include="System.ServiceModel.Http" />
 		<!-- CVE-2023-29331 -->
```

Comment:
Same as above for the upgrade comments, do we want to keep the `CVE` comments in the project files with the dependency, or should they be in the `Directory.Packages.props` file with the version?

Reply:
Moved the comment